cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1,24 +1,269 @@
+            "#});
+
+            table
+        };
+
+        install_table.decor_mut().set_prefix(indoc! {r#"
+            #
+            # This is a Flox environment manifest.
+            # Visit flox.dev/docs/concepts/manifest/
+            # or see flox-edit(1), manifest.toml(1) for more information.
+            #
+
+            # List packages you wish to install in your environment inside
+            # the `[install]` section.
+        "#});
+
+        manifest.insert(MANIFEST_INSTALL_TABLE_KEY, Item::Table(install_table));
+
+        // `[vars]` table
+        let mut vars_table = Table::new();
+
+        vars_table.decor_mut().set_prefix(indoc! {r#"
+            # Set environment variables in the `[vars]` section. These variables may not
+            # reference once another, and are added to the environment without first
+            # expanding them. They are available for use in the `[profile]` and `[hook]`
+            # scripts.
+        "#});
+        vars_table.decor_mut().set_suffix(indoc! {r#"
+
+            # message = "Howdy"
+        "#});
+
+        manifest.insert(MANIFEST_VARS_TABLE_KEY, Item::Table(vars_table));
+
+        // `[hook]` table
+        let mut hook_table = Table::new();
+
+        if let Some(ref hook_on_activate_script) = customization.hook_on_activate {
+            let on_activate_content = formatdoc! {r#"
+                """
+                {}
+                """"#
+            , indent::indent_all_by(2, hook_on_activate_script)};
+
+            hook_table.insert("on-activate", value(on_activate_content));
+        } else {
+            hook_table.decor_mut().set_suffix(indoc! {r#"
+
+                # on-activate = """
+                #     # Set variables, create files and directories
+                #     venv_dir="$(mktemp -d)"
+                #     export venv_dir
+                #
+                #     # Perform initialization steps, e.g. create a python venv
+                #     python -m venv "$venv_dir"
+                #
+                # """
+            "#});
+        };
+
+        hook_table.decor_mut().set_prefix(indoc! {r#"
+            # The `hook.on-activate` script is run by the *bash* shell immediately upon
+            # activating an environment, and will not be invoked if Flox detects that the
+            # environment has previously been activated. Variables set by the script will
+            # be inherited by `[profile]` scripts defined below. Note that any stdout
+            # generated by the script will be redirected to stderr.
+        "#});
+
+        manifest.insert(MANIFEST_HOOK_TABLE_KEY, Item::Table(hook_table));
+
+        // `[profile]` table
+        let mut profile_table = Table::new();
+
+        match customization {
+            InitCustomization {
+                profile_common: None,
+                profile_bash: None,
+                profile_zsh: None,
+                ..
+            } => {
```

Comment:
suggestion (nb): handle these individually, i.e. keep other profile scripts documented even if only one is used.
The counterargument is that it adds unnecessary noise

I'll check if there is a preference, for now we can keep it as is.

Reply:
Yeah I'd prefer to just have `common` in there, we're about to add two more shells and I don't think we want this comment to grow with each shell we add